catalyst/dl/runner/__init__.py

```diff
@@ -1,3 +1,4 @@
 # flake8: noqa
 
 from .supervised import SupervisedRunner
+from .wandb import *
```

Comment:
also

Reply:
maybe it's better to move this
```
try:
    import wandb
    WANDB_ENABLED = True
except ImportError:
    logger.warning(
        "wandb not available, switching to pickle. "
        "To install wandb, run `pip install wandb`."
    )
    WANDB_ENABLED = False

if WANDB_ENABLED:
    from .wandb import WandbRunner
```
here to simplify `catalyst/dl/runner/wandb.py` code?